lib/Crypto/Random/random.py

```diff
@@ -67,9 +67,7 @@ def randrange(self, *args):
         if step == 0:
             raise ValueError("randrange step argument must not be zero")
 
-        num_choices = ceil_div(stop - start, step)
-        if num_choices < 0:
-            num_choices = 0
+        num_choices = max(ceil_div(stop - start, step), 0)
```

Comment:
This is enough, because anything less than 1 causes the same error below:
```suggestion
        num_choices = ceil_div(stop - start, step)
```